src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -26,6 +26,11 @@ public abstract partial class Diagnostic : IEquatable<Diagnostic?>, IFormattable
         /// </summary>
         internal const int DefaultWarningLevel = 4;
 
+        /// <summary>
+        /// The warning level used for hidden and info diagnostics.
```

Comment:
Nit: can you add a more complete description? IE, why this is different from Default

Reply:
```suggestion
        /// The warning level used for hidden and info diagnostics. Because these diagnostics interact with other editor features, we want them to always be produced unless /warn:0 is set.
```